too_simple_server/tasks/main.yml

```diff
@@ -1,4 +1,21 @@
 ---
+- name: Get latest version
+  script: get_latest_release.sh
+  register: LATEST_VERSION
+
+- name: Check active version of too_simple_server
+  shell: "{{ server|quote }} --version"
+  register: CURRENT_VERSION
+  ignore_errors: yes
+
+- become: yes
+  block:
+    - name: Stop server
+      include_tasks: stop_server.yml
+    - name: Start server
+      include_tasks: start_server.yml
+  when: CURRENT_VERSION.stdout_lines[0] == LATEST_VERSION.stdout_lines[0] and CURRENT_VERSION.rc == 0
```

Comment:
`CURRENT_VERSION.stdout_lines[0] == LATEST_VERSION.stdout_lines[0]`
Those two should be better saved as variables, like `current_version` and `latest_version`